### PR TITLE
requirements: Update flattentool

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

File: cove_iati/tests_functional.py
Comment (simon-20) [inline cove_iati/tests_functional.py:1]: Just curious as to why this custom namespace has moved here?

**Review by simon-20 — APPROVED** [review]

File: cove_iati/tests_functional.py
Comment (Bjwebb) [reply cove_iati/tests_functional.py:1]: This is a result of handling namespaces that do/don't cover a whole activity: https://github.com/OpenDataServices/flatten-tool/commit/7e12343ff6735a6c4ddc231519304cb153d40e6c

The easiest way of implementing it was to create all parent elements after their children, and pass the namespace information to the parents. lxml then outputs the namespaces in the most "efficient" places possible. A side effect of this is that namespaces that were on the activity level are now on the root element.

Comment (simon-20) [reply cove_iati/tests_functional.py:1]: Ah, that's helpful, thanks.

**Comment by Bjwebb** [comment]
I will merge/deploy this on Tuesday.

**Pull request merged**